tst/EnergyPlus/unit/HVACDXSystem.unit.cc

```diff
@@ -801,4 +802,216 @@ TEST_F(EnergyPlusFixture, VariableSpeedCoils_LatentDegradation_Test)
     EXPECT_NEAR(0.109, state->dataVariableSpeedCoils->VarSpeedCoil(1).PartLoadRatio, 0.001); // PLR is lower, latent capacity is 0
 }
 
+TEST_F(EnergyPlusFixture, NewDXCoilModel_RHControl)
+{
+    // issue #6920
+    std::string const idf_objects = delimited_string({
```

Comment:
Can these be `string_view` instead of `string`?

Reply:
> Can these be `string_view` instead of `string`?

bool EnergyPlusFixture::process_idf(std::string const &idf_snippet, bool use_assertions)

I guess, but this is the unit test syntax for now.

Reply:
That would be a great thing to add, we'd just need to overload the process_idf function to take a string view version.